src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataInternalLatestIT.java

```diff
@@ -71,173 +71,174 @@ public void testGetMetaDataUseConnectionCtx() throws SQLException {
-    assertEquals(0, resultSet.getInt("CHAR_OCTET_LENGTH"));
-    assertEquals(2, resultSet.getInt("ORDINAL_POSITION"));
-    assertEquals("", resultSet.getString("IS_NULLABLE"));
-    assertEquals(
-        "FUNC112() RETURN TABLE (COLA VARCHAR, COLB NUMBER, BIN2 BINARY, SHAREDCOL NUMBER)",
-        resultSet.getString("SPECIFIC_NAME"));
-    resultSet.next();
-    assertEquals("JDBC_DB1", resultSet.getString("FUNCTION_CAT"));
-    assertEquals("JDBC_SCHEMA11", resultSet.getString("FUNCTION_SCHEM"));
-    assertEquals("FUNC112", resultSet.getString("FUNCTION_NAME"));
-    assertEquals("BIN2", resultSet.getString("COLUMN_NAME"));
-    assertEquals(DatabaseMetaData.functionColumnResult, resultSet.getInt("COLUMN_TYPE"));
-    assertEquals(Types.BINARY, resultSet.getInt("DATA_TYPE"));
-    assertEquals("BINARY", resultSet.getString("TYPE_NAME"));
-    assertEquals(38, resultSet.getInt("PRECISION"));
-    assertEquals(0, resultSet.getInt("LENGTH"));
-    assertEquals(0, resultSet.getInt("SCALE"));
-    assertEquals(10, resultSet.getInt("RADIX"));
-    assertEquals(DatabaseMetaData.functionNullableUnknown, resultSet.getInt("NULLABLE"));
-    assertEquals("returns table of 4 columns", resultSet.getString("REMARKS"));
-    assertEquals(8388608, resultSet.getInt("CHAR_OCTET_LENGTH"));
-    assertEquals(3, resultSet.getInt("ORDINAL_POSITION"));
-    assertEquals("", resultSet.getString("IS_NULLABLE"));
-    assertEquals(
-        "FUNC112() RETURN TABLE (COLA VARCHAR, COLB NUMBER, BIN2 BINARY, SHAREDCOL NUMBER)",
-        resultSet.getString("SPECIFIC_NAME"));
-    resultSet.next();
-    assertEquals("JDBC_DB1", resultSet.getString("FUNCTION_CAT"));
-    assertEquals("JDBC_SCHEMA11", resultSet.getString("FUNCTION_SCHEM"));
-    assertEquals("FUNC112", resultSet.getString("FUNCTION_NAME"));
-    assertEquals("SHAREDCOL", resultSet.getString("COLUMN_NAME"));
-    assertEquals(DatabaseMetaData.functionColumnResult, resultSet.getInt("COLUMN_TYPE"));
-    assertEquals(Types.NUMERIC, resultSet.getInt("DATA_TYPE"));
-    assertEquals("NUMBER", resultSet.getString("TYPE_NAME"));
-    assertEquals(38, resultSet.getInt("PRECISION"));
-    assertEquals(0, resultSet.getInt("LENGTH"));
-    assertEquals(0, resultSet.getInt("SCALE"));
-    assertEquals(10, resultSet.getInt("RADIX"));
-    assertEquals(DatabaseMetaData.functionNullableUnknown, resultSet.getInt("NULLABLE"));
-    assertEquals("returns table of 4 columns", resultSet.getString("REMARKS"));
-    assertEquals(0, resultSet.getInt("CHAR_OCTET_LENGTH"));
-    assertEquals(4, resultSet.getInt("ORDINAL_POSITION"));
-    assertEquals("", resultSet.getString("IS_NULLABLE"));
-    assertEquals(
-        "FUNC112() RETURN TABLE (COLA VARCHAR, COLB NUMBER, BIN2 BINARY, SHAREDCOL NUMBER)",
-        resultSet.getString("SPECIFIC_NAME"));
-    resultSet = databaseMetaData.getFunctionColumns(null, "%", "%", "%");
-    // we have 81 columns returned
-    assertEquals(4, getSizeOfResultSet(resultSet));
```

Comment:
remove this. this result can differ depending on which account is used to run the test.